src/jit/morph.cpp

```diff
@@ -3739,21 +3755,19 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
             if (!lateArgsComputed)
             {
                 // This is a register argument - put it in the table
-                fgArgTabEntryPtr newArg = call->fgArgInfo->AddRegArg(argIndex, 
-                                                                     argx, 
-                                                                     args, 
-                                                                     nextRegNum, 
-                                                                     size, 
-                                                                     argAlign
+                fgArgTabEntryPtr argEntry = call->fgArgInfo->AddRegArg(argIndex,
+                                                                       argx, 
+                                                                       args, 
+                                                                       nextRegNum, 
+                                                                       size, 
+                                                                       argAlign
 #if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
                                                                      , isStructArg, 
-                                                                     nextOtherRegNum, 
-                                                                     &structDesc
+                                                                       nextOtherRegNum, 
+                                                                       &structDesc
 #endif // defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-                    );
-                (void)newArg; //prevent "unused variable" error from GCC
-
-                newArg->SetIsHfaRegArg(passUsingFloatRegs && isHfaArg); // Note on Arm32 a HFA is passed in int regs for varargs
+                                                                    );
+                argEntry->SetIsHfaRegArg(passUsingFloatRegs && isHfaArg); // Note on Arm32 a HFA is passed in int regs for varargs
 
 #ifdef _TARGET_ARM_
                 newArg->SetIsBackFilled(isBackFilled);
```

Comment:
@briansull Shouldn't newArg be changed to argEntry here as well? I see a
> error: use of undeclared identifier 'newArg'
>      newArg->SetIsBackFilled(isBackFilled);
>      ^

error on ARM builds.